Microsoft.WindowsAzure.Storage/src/resources.cpp

```diff
@@ -20,10 +20,10 @@
 
 namespace azure { namespace storage { namespace protocol {
 
-#define _RECOURSES
+#define _RESOURCES
 #define DAT(a, b) const char* a = b;
```

Comment:
Can you raise another PR for this fix.  We recommend one PR for just one purpose.